service/frontend/dcRedirectionHandler.go

```diff
@@ -1670,6 +1670,34 @@ func (handler *DCRedirectionHandlerImpl) UpdateWorkflow(
 	return resp, err
 }
 
+func (handler *DCRedirectionHandlerImpl) DescribeBatchOperation(
+  ctx context.Context,
+  request *workflowservice.DescribeBatchOperationRequest,
+) (resp *workflowservice.DescribeBatchOperationResponse, retError error) {
+  panic("DCRedirectionHandlerImpl.DescribeBatchOperationRequest not implemented")
```

Comment:
```suggestion
  return nil, serviceerror.NewUnimplemented("DescribeBatchOperationRequest not implemented")
```
instead of panic everywhere